### PR TITLE
Quotes cleanup

### PR DESCRIPTION
**Pull request by ghost**
' to "

### CODE REVIEW

File: adsb_docker_install.sh [hunk 1]
```diff
@@ -1473,13 +1473,13 @@ function create_docker_compose_yml_file() {
             echo "    depends_on:"
             echo "      - readsb"
             echo "    environment:"
-            echo '      - ALT=${FEEDER_ALT_M}m'
+            echo "      - ALT=${FEEDER_ALT_M}m"
```
Comment (mikenye) [inline adsb_docker_install.sh:1476]: These quotes need to be single quotes, as the intention here for the `docker-compose.yml` file to use variables from the `.env` file, not variables from the script.

**Review by mikenye — CHANGES_REQUESTED** [review]
Hi @natur3kid,

Thanks so much for helping with this! :-)

The changes on lines 932 and 941 are valid, thanks for this.

The remaining changes in the file are unfortunately not valid. The intention is to actually write `${VARIABLE}` into the `docker-compose.yml` file, so variables from the `.env` file will be used.

I should've commented this better, so I apologise that you've spent time fixing this.

Do you want to reverse the changes from line 1476 onwards, or should I just cancel the PR?

I'll leave it up to you.

Thanks again.

File: adsb_docker_install.sh (hunk 1 above)
Comment (ghost) [reply adsb_docker_install.sh:1476]: Ah! Okay, still learning. Just cancel it, I was just eager to help, but it is okay to cancel :-) Just change 932 and 941 then.

**Comment by ghost** [comment]
ok

**Pull request closed without merging**